src/ES.FX.Ignite.Spark/SparkGuard.cs

```diff
@@ -5,11 +5,15 @@ namespace ES.FX.Ignite.Spark;
 
 public static class SparkGuard
 {
+    public static string GuardSparkDefaultConfigurationErrorMessageGenerator(string key)
```

Comment:
Can you rename this to something simpler? It contains redundant references to the spark guard